tools/crm_shadow.c

```diff
@@ -320,7 +315,7 @@ main(int argc, char **argv)
             case '$':
             case '?':
                 pcmk__cli_help(flag, CRM_EX_OK);
-                break;
+                goto done;
```

Comment:
not needed -- I forgot `pcmk__cli_help()` exits

Reply:
Forgot to revert this in the most recent push, but it doesn't affect the actual behavior, and the second (GOption) commit gets rid of this whole block, so I'm not worried about it.